spacepy/pybats/bats.py

```diff
@@ -2081,26 +2208,25 @@ def add_b_magsphere_legacy(self, target=None, loc=111, style='mag',
         outwards away from the furthest point from the last traced and
         closed field line.  This is repeated until open lines are found.
         '''
-        
-        import matplotlib.pyplot as plt
+
         from matplotlib.collections import LineCollection
-        from numpy import (arctan, cos, sin, where, pi, log, 
+        from numpy import (arctan, cos, sin, where, pi, log,
```

Comment:
Again. I think we should be minimizing repeated and/or in-function imports. It's also nice to be clearer that it's numpy on calling by using `np.where`, for example.